### PR TITLE
[#2897] Fix test-timer to proxy attributed of the wrapped method.

### PR DESCRIPTION
**Pull request by adiroiban**
Problem
======

When test-timer is used to measure the exact test time (without setup/teardown) it wraps the test method.

This wrapper does not export the attributes attached to the wrapped method

Changes
=======

Use a smarter wrapper which proxies the attributes.

As a drive by I have updated the release notes of an older version.

How to test
=========

reviewers: @alibotean 

check that changes make sense.

run test with the test timer plugin:

`paver test normal --with-timer`

To fix this I wrote the test first... so you can revert the changed from the main code and keep the tests.

When you run the test with `--with-timer` the test should fail with the old code and pass with the new code.

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

**Comment by adiroiban** [comment]
I have also released 0.34.5 since chevah.server is not yet ported to py3 compatibiliyt and tests are failing with the new impirical.

File: release-notes.rst [hunk 1]
```diff
@@ -2,6 +2,24 @@ Release notes for chevah.empirical
 ==================================
 
 
+0.35.1 - 08/07/2015
+-------------------
+
+* Fix timer nose plugin to proxy attributes of the timed method..
```
Comment (alibotean) [inline release-notes.rst]: Double .

**Comment by alibotean** [comment]
All good.

changes-approved

**Pull request closed without merging**